include/mbedtls/ssl.h

```diff
@@ -48,6 +48,11 @@
 #include "ecdh.h"
 #endif
 
+#if defined(MBEDTLS_USE_UECC)
```

Comment:
A note on commit structure: I think it would have been better to have separate commits for (1) import of external code and modification of build system (2) new option in config.h (3) edits to existing modules in the library (split by feature). That's just something to keep in mind for future commits, no need to rewrite the history here (we usually avoid rewriting history while review is in progress).

Reply:
Noted